### PR TITLE
Created doc: LIMIT

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+```nGQL
+LIMIT [<offset>,] <number_rows>
+```
+
+|Parameter|Description|
+|-|-|
+|`offset`|Optional. It specifies the number of rows to be skipped. The offset starts from zero.|
+|`number_rows`|It specifies the number of rows to be returned. It can be an expression that outputs a non-negative integer.|
+
+> **NOTE:** In a `MATCH` statement, you can use a `LIMIT` clause with or without a pipe symbol, but in other statements, you must add a pipe symbol before the `LIMIT` clause.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:16]: good catch

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| "Kyle Anderson"         | 25  |
++-------------------------+-----+
+| "Joel Embiid"           | 25  |
++-------------------------+-----+
+| "Kyrie Irving"          | 26  |
++-------------------------+-----+
+Got 5 rows (time spent 2922/3854 us)
+```
+
+> **NOTE:** To use a `LIMIT` offset in `MATCH`, you must add a pipe before `LIMIT`.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:68]: good catch, pig joe

Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:68]: hmmmm...

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+You can also use [`SKIP`](./skip.md) to skip the top M rows.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:70]: 404

File: mkdocs.yml [hunk 4]
```diff
@@ -123,6 +123,7 @@ nav:
       
       - Clauses and options:
           - ORDER BY: 3.ngql-guide/8.clauses-and-options/order-by.md
+          - LIMIT: 3.ngql-guide/8.clauses-and-options/limit.md
```
Comment (Amber1990Zhang) [inline mkdocs.yml:126]: nice

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md (hunk 3 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:70]: It'll be ready soon. Maybe tomorrow.

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+You can use an expression to specify the number of rows to be returned. The expression must output a non-negative integer.
+
+> **NOTE:** Fraction expressions composed of two integers are automatically floored to integers. For example, 8/6 is floored to 1.
+
+```nGQL
+nebula> MATCH (v:player) \
+        RETURN v.name AS Name, v.age AS Age \
+        ORDER BY Age \
+        LIMIT rand32(5)
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:82]: ;

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+| "Ben Simmons"           | 22  |
++-------------------------+-----+
+| "Kristaps Porzingis"    | 23  |
++-------------------------+-----+
+| "Giannis Antetokounmpo" | 24  |
++-------------------------+-----+
+Got 4 rows (time spent 2621/3343 us)
+```
+
+> **NOTE:** For now, only in `MATCH` statements can you use an expression with `LIMIT`. And you must omit the pipe symbol.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:97]: good catch

**Review by Amber1990Zhang — DISMISSED** [review]
nice! Minors

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**